programs/fund-sender/src/utils/state.rs

```diff
@@ -89,3 +93,25 @@ pub struct SendFund<'info> {
     pub destination_account: UncheckedAccount<'info>,
     pub system_program: Program<'info, System>,
 }
+
+#[derive(Accounts)]
+pub struct StoreCertificates<'info> {
+    // to send the received retired climate token to a hold account
+    #[account(mut)]
+    pub payer: Signer<'info>,
+    pub state: Account<'info, State>,
+    #[account(
+        mut,
+        seeds = [OUTPUT_YIELD_ACCOUNT, &state.destination_seed, state.key().as_ref()],
+        bump = state.output_yield_account_bump,
+    )]
+    /// CHECK: Must be correctly derived from the state
+    pub output_yield_account: UncheckedAccount<'info>,
+    #[account(mut)]
+    /// CHECK: Must be correctly derived from the state
```

Comment:
I would remove this comment and replace it with something like:
```suggestion
    ///  A token account owned by the outputYieldAccount
```
and then I would add the constraint:
#[account(token::authority = output_yield_account  @ SomeError)]

Actually, you don't need that constraint, as the token program checks it already, so if we wanted to save a couple of compute units, we could leave it out, but IMO it adds clarity.